data/remnant missions.txt

```diff
@@ -110,13 +179,12 @@ mission "Remnant: Defense 1"
 event "remnant defense timer"
 	clear "Remnant defense delay"
 
-
-
+			
```

Comment:
the diff doesn't show it, but it seems there are some stray tabs in here

Reply:
Thanks, done.